website/pages/docs/drivers/external/lxc.mdx

```diff
@@ -71,7 +71,11 @@ The `lxc` driver supports the following configuration in the job spec:
   }
   ```
 
-- `volumes` - (Optional) A list of `host_path:container_path` strings to bind-mount host paths to container paths. Mounting host paths outside of the allocation directory can be disabled on clients by setting the [`volumes_enabled`](#volumes_enabled) option set to false. This will limit volumes to directories that exist inside the allocation directory.
+- `volumes` - (Optional) A list of `host_path:container_path` strings to
+bind-mount host paths to container paths. Mounting host paths outside of the
+[allocation working directory] can be disabled on clients by setting the
+[`volumes_enabled`](#volumes_enabled) option set to false. This will limit
+volumes to directories that exist inside the allocation directory.
```

Comment:
This is a WAG assuming that LXC's default harmonizes with docker.  Toss if not.
```suggestion
- `volumes` - (Optional) A list of `host_path:container_path` strings to
  bind-mount host paths to container paths. Mounting host paths outside of the
  [allocation working directory] is disabled by default and limits volumes to
  directories that exist inside the allocation working directory. You can enable
  mounting host paths outside of the [allocation working directory] on
  individual clients by setting the [`volumes_enabled`](#volumes_enabled) option
  to `true` in the client's configuration```

Reply:
It most likely does... but let me check their code to confirm that.

Reply:
No, unfortunately it's backwards. That driver defaults `volumes_enabled` to true: https://github.com/hashicorp/nomad-driver-lxc/blob/master/lxc/driver.go#L47. But I'll use your better wording here and swap the values.

Reply:
Fixed in 299b5b1